tools/c7n_sentry/c7n_sentry/c7nsentry.py

```diff
@@ -333,7 +333,7 @@ def get_function(session_factory, name, handler, role,
     config = dict(
         name=name,
         handler=handler,
-        runtime='python2.7',
+        runtime='python2.7' if str is bytes else 'python3.6',
```

Comment:
ditto re explicit runtime config.

Reply:
Per https://github.com/capitalone/cloud-custodian/pull/1417#discussion_r135745163, I think the default needs to be the _only_ option: to upload whatever the local runtime is, so that bundled deps are as expected. Otherwise we end up trying to run py27 code under py36 and vice versa, which is lucky to work in more than trivial cases.

Reply:
the issue is that we might have multiple users running custodian against the same account. the source code is the same, and already tested against 2 and 3. what's odd is switching out depending on who is running it in and on what environment.. explicit vs implicit.

Reply:
What if we compare existing remote and local runtimes and throw an error if they are different?

Reply:
Exploring in https://github.com/whit537/cloud-custodian/pull/40 ...

Reply:
PR started in https://github.com/capitalone/cloud-custodian/pull/1546 w/ demonstrative wip.